presto-common/src/main/java/com/facebook/presto/common/type/LongEnumType.java

```diff
@@ -72,12 +81,15 @@ public String getDisplayName()
     public static class LongEnumMap
     {
         private final Map<String, Long> enumMap;
+        private Map<Long, String> flippedEnumMap;
```

Comment:
Have we considered using a `BiMap`? e.g. `ImmutableBiMap`.

Reply:
I tried that but that requires adding a maven dependency, and since this code lives in presto-common, that is not desirable